Backend/app/rider-platform/rider-app/Main/src/API/UI/Select.hs

```diff
@@ -83,14 +86,16 @@ select personId estimateId = withFlowHandlerAPI . withPersonIdLogTag personId $
   let autoAssignFlag = False
   let req = DSelect.DEstimateSelectReq {customerExtraFee = Nothing, autoAssignEnabled = autoAssignFlag, autoAssignEnabledV2 = Nothing}
   dSelectReq <- DSelect.select personId estimateId req
-  becknReq <- ACL.buildSelectReq dSelectReq
+  estimate <- Estimate.findById estimateId >>= fromMaybeM (EstimateDoesNotExist estimateId.getId)
+  searchRetry <- SearchRetry.findById $ cast estimate.requestId
```

Comment:
these queries should be part of `DSelect.select` function

Reply:
done